specification/network/resource-manager/Microsoft.Network/stable/2019-07-01/virtualRouter.json

```diff
@@ -0,0 +1,699 @@
+            "schema": {
+              "$ref": "#/definitions/VirtualRouterPeering"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Virtual Router Peering": {
+            "$ref": "./examples/VirtualRouterPeeringPut.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualRouters/{virtualRouterName}/peerings": {
+      "get": {
+        "tags": [
+          "VirtualRouterPeerings"
+        ],
+        "operationId": "VirtualRouterPeerings_List",
+        "description": "Lists all Virtual Router Peerings in a Virtual Router resource.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "virtualRouterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Virtual Router."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The operation returns a list of Virtual Router Peering resources.",
+            "schema": {
+              "$ref": "#/definitions/VirtualRouterPeeringListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all Virtual Router Peerings for a given Virtual Router": {
+            "$ref": "./examples/VirtualRouterPeeringList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "VirtualRouter": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/VirtualRouterPropertiesFormat",
+          "description": "Properties of the Virtual Router."
+        },
+        "etag": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Gets a unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/Resource"
+        }
+      ],
+      "description": "VirtualRouter Resource."
+    },
+    "VirtualRouterPropertiesFormat": {
+      "properties": {
+        "virtualRouterAsn": {
+          "type": "integer",
+		  "readOnly": false,
+          "format": "int64",
```

Comment:
tabs -> spaces   (before "readonly" property of virtualRouterAsn)

Reply:
same after "maximum" property

---
In reply to: [317344838](https://github.com/Azure/azure-rest-api-specs/pull/7005#discussion_r317344838) [](ancestors = 317344838)